### DISCUSSION TITLE
O que utilizar com React?

### DISCUSSION THREAD

**Discussion by francysreymer** [frontendbr/forum#1940] [Discussão]
Pessoal, no Vue eu costumo utilizar uma biblioteca chamada Vuetify que é muito boa e completa com todos os componentes que preciso. No react, tem algo parecido? O que vocês recomendam?
Fora isso, tem algum boilerplate que vcs gostam e utilizam bastante?

**Comment by victorferreira** [comment]
O que não falta por aí é opção de biblioteca de componentes em React. Eu já trabalhei com [Material-UI](https://material-ui.com/) e [Ant Design](https://ant.design/) e dificilmente você não vai encontrar o que precisa nelas. Outra opção que parece ser bem interessante é o [Chackra](https://chakra-ui.com/),  mas eu nunca cheguei a usar.

**Comment by wilsonneto-swe** [comment]
Fala @francysreymer, assim como o @victorferreira falou, libs de UI é o que não falta rs Sobre boilerplarte, nos sites das libs você já consegue pegar alguns "esqueletos" prontos para o start inicial.

 Adicionando mais algumas ao comentário acima:
[React Bootstrap](https://react-bootstrap.github.io/components/alerts)
[Fluent UI](https://developer.microsoft.com/en-us/fluentui#/) (Se quiser uma carinha de Outlook/Office)

E para RN tem a [RN Paper](https://reactnativepaper.com/), muito boa também.

Abraços!

**Comment by ninetails** [comment]
Tem um site com [uma coleção bem maneira](https://bit.dev/collections) que é possível instalar cada componente individualmente (claro, também é bom ter em mente que estas podem ter dependências de outras bibliotecas).

Acho que o que mais deve-se ter atenção é se estas dependências fazem sentido dentro da tua stack.
